listenbrainz/webserver/static/js/src/PinRecordingModal.tsx

```diff
@@ -59,6 +59,7 @@ export default class PinRecordingModal extends React.Component<
             `${recordingToPin.track_metadata.artist_name} - ${recordingToPin.track_metadata.track_name}`
           );
           this.setState({ blurbContent: "" });
+          window.location.reload();
```

Comment:
let's add a comment explaining why we reload here, it's not immediately obvious to someone who doesn't have much context on the code